src/pages/steps/AdditionalInformationStep/PriceInformation.tsx

```diff
@@ -407,11 +408,24 @@ const PriceInformation = ({
           </Inline>
         </Stack>
         <Stack spacing={4}>
-          <Alert fullWidth>
+          <Alert
+            fullWidth
+            css={`
+              width: 100%;
+            `}
+            variant={AlertVariants.PRIMARY}
+          >
             {t('create.additionalInformation.price_info.global_info')}
           </Alert>
-          {hasUitpasPrices && (
-            <Alert variant={AlertVariants.PRIMARY} marginBottom={3}>
+          {true && (
```

Comment:
```suggestion
          {(
```
This seems unnecessary 😛